<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выживаемость народа</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelPeople">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите название народа</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumberOfPeople">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите количество людей</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберить уровень развития</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemperature">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите температуру из предложенных</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTerrainConditions">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>55</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите условия местности</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFoodCrop">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите продовольственную культуру</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPeople">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Лесные эльфы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalculate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>740</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать, выживет ли народ при данных условиях</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxNumberOfPeople">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>55</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLevel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTemperature">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTerrainConditions">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>55</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFoodCrop">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionLevelsDescription"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionLevelsDescription">
   <property name="text">
    <string>Описание уровней</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе Выживаемость народа…</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
